agent/util-scripts/pbench-register-tool

```diff
@@ -115,6 +115,7 @@ function usage() {
 	printf -- "\tof the file should contain a host name, optionally followed by a label\n"
 	printf -- "\tseparated by a comma (\",\"); empty lines are ignored, and comments are\n"
 	printf -- "\tdenoted by a leading hash, or pound (\"#\"), character.\n"
+	printf -- "\n\t--persistent or --transient can be used to specify tool run type.\n"
 	printf -- "\nAvailable tools:\n"
 	local tool=""
 	python3 -c "import sys, json; meta = json.load(open(sys.argv[1])); [print(f'\t{tool}') for tool in (*meta['transient'].keys(), *meta['persistent'].keys()) ]" ${pbench_bin}/tool-scripts/meta.json
```

Comment:
This would mix all the persistent and transient tools together, though to be fair, there is currently nothing good indicating what is what, so I agree a change is needed. I could change it to output:

Available Tools:
-Transient:
    <tool>
    <tool>
-Persistent:
    <tool>
    <tool>
    
    or something along those lines... @portante thoughts?

Reply:
I would be good with either approach.

(I thought about suggesting adding "headers", but didn't want to muddy the waters....)

[FWIW, I think Peter is on hiatus wrt reviewing Server-side code, so, if you really want his opinion, I think you'll need to ping him harder than just an @mention.]

If you would prefer to add headers rather than sorting the lists together, I say go for it.

Reply:
> [FWIW, I think Peter is on hiatus wrt reviewing Server-side code, so, if you really want his opinion, I think you'll need to ping him harder than just an @mention.]

You knew this, right? For posterity, this is agent code.

> If you would prefer to add headers rather than sorting the lists together, I say go for it.

Yeah, making this sort of display unambiguously clear and straightforward isn't easy; I'm OK with leaving it as it is.

Reply:
> For posterity, this is agent code.

LOL...so it is!  I've been doing so much back-and-forth between reviews that I forgot where I was.  :-(